README.md

```diff
@@ -1,63 +1,12 @@
-
-Ist bereits eine Liste aller Messdiener in einem Tabellenformat vorhanden, so kann der Import dieser als CSV-Datei die Dateneingabe enorm beschleunigen, ebenso das Verwenden von Serienterminen für periodische Termine.
-
-Ausführliche Informationen zu Bedienung und Funktionsweise finden sich im [Handbuch](Handbuch.pdf)
-
-## Daten
-
-* Version: 2.0.4
-* Größe: 7.2 MB
-* System-Voraussetzungen: Windows ab XP (32/64 bit), Linux und Mac auf Nachfrage
-* Autor: Yannik Schälte
-
-## Download
-
-Die Verwendung des Programms ist kostenlos möglich. Rückmeldungen und Kritik sind sehr willkommen, um das Programm weiter verbessern zu können. Suchen Sie sich einfach eine der folgenden gleichwertigen Installationsvarianten aus:
-
-* [MiniPlaner.zip](MiniPlaner.zip) (heruntergeladene Datei per Rechtsklick + "Alle Extrahieren" entpacken
-* [MiniPlaner.7z](MiniPlaner.7z) (mit dem Kompressionsprogramm 7-Zip entpacken
-* [MiniPlaner_Setup.exe](MiniPlaner_Setup.exe) (Installer starten und Anweisungen folgen)
-
-Im entpackten Verzeichnis ``MiniPlaner/`` befindet sich die ausführbare Datei ``MiniPlaner.exe``, dies ist das zu startende Hauptprogramm. Auch eine Deinstallation ist denkbar einfach: Löschen Sie einfach das Verzeichnis ``MiniPlaner/``, da das Programm lediglich hier Daten speichert.
-
-## Beispiel
-
-Folgende Downloads sind bereits mit Beispieldaten gefüllt und können temporär neben der eigentlichen Installation verwendet werden (eine Installation verwaltet stets nur einen Datensatz):
-
-* [MiniPlaner_Beispiel.zip](MiniPlaner_Beispiel.zip)
-* [MiniPlaner_Beispiel.7z](MiniPlaner_Beispiel.7z)
-
-## Add-Ons
-
-* **MiniInterface**: Deponieren Sie die Datei [MiniInterface.php](MiniInterface.php) auf Ihrer Gemeinde-Homepage, so können Messdiener über ihren Browser zuhause selbst eintragen, wann sie dienen möchten. Diese Daten können Sie dann im **MiniPlaner** bei der Planerstellung automatisch berücksichtigen lassen.
-
-## Updates
-
-Wenn Sie eine ältere Version (&le; 2.0.3) des Programms bereits installiert haben, ersetzen Sie im Verzeichnis ``MiniPlaner/`` die Datei ``MiniPlaner.exe`` einfach durch [diese](MiniPlaner.exe) aktuelle Version.
-
-* 01.03.2019 (2.0.4): move project to github.com
-* 12.10.2016 (2.0.3): several minor error fixes, added "Termin::extras" button and "Mini::bem" attribute, added fill-in-help
-* 30.04.2016 (2.0.2): integrated MiniInterface, certain changes of mass data in plan window possible
-* 18.02.2016 (2.001): 2 minor error fixes, added "Mini::extras" button
-* 11.10.2015 (2.0): initial release
-
-## Hilfe und Kontakt
-
-* [Handbuch](Handbuch.pdf)
-* E-Mail: ``yannik.schaelte <at> gmail.com``
-
-
-&copy; 2018 Yannik Schälte. Alle Rechte vorbehalten.
+If compiling in MSYS2, see the package scripts in [MiniPlaner-MINGW](https://github.com/mwkroening/MiniPlaner-MINGW) for details.
```

Comment:
It's just some info, right? One could add that directly in here.

Reply:
Actually it's more than that.

Those are package scripts for MSYS2. Those are the preferred method of building releases. Currently, they reside in the `impl` branch of that repository, as they depend on this to be merged first.

Package scripts should be separate from the project itself as they are specific to the distribution. Once a new release of MiniPlaner has been issued, I'll publish a package script in the AUR for example.

Reply:
Would you rather have this repository in your GitHub namespace?

Reply:
Ah ok. Then it of course makes sense to keep things in a different repository. If it's about distribution-specificity only, one could also create corresponding branches, I don't have experience on that though.

No need to move it to my namespace I think.

Reply:
It makes sense to have those as separate projects for two reasons:
* The build scripts don't share any code with the project code.
* The build scripts are being maintained by whomever is feeling responsible for that platform. Fedora users could maintain RPMs, Debian users could maintain debs, I could maintain the Arch and MSYS2 packages, as they actually use the same package manager.

The project would provide a ready-to-use distribution for Windows (based on my MSYS2 package scripts). We could also look into providing official [flatpaks](https://flatpak.org/), [snaps](https://snapcraft.io/) or [AppImages](https://appimage.org/) in the future. In theory, we could provide distribution specific packages as well, but I'd rather leave that task to users of the corresponding distribution.

Reply:
> It makes sense to have those as separate projects for two reasons:
> 
> * The build scripts don't share any code with the project code.
> * The build scripts are being maintained by whomever is feeling responsible for that platform. Fedora users could maintain RPMs, Debian users could maintain debs, I could maintain the Arch and MSYS2 packages, as they actually use the same package manager.
> 
> The project would provide a ready-to-use distribution for Windows (based on my MSYS2 package scripts). We could also look into providing official [flatpaks](https://flatpak.org/), [snaps](https://snapcraft.io/) or [AppImages](https://appimage.org/) in the future. In theory, we could provide distribution specific packages as well, but I'd rather leave that task to users of the corresponding distribution.

I think probably 95% of users are on Windows, so a standalone Windows distro should suffice for most, other distros would of course be great, however I'm not aware of anyone except us using it. Links to the different distros could then also be put on the gh-pages.